test/test_dof_transformations.py

```diff
@@ -9,7 +9,7 @@
 from .utils import parametrize_over_elements
 
 
-@parametrize_over_elements(3)
+@parametrize_over_elements(1)
```

Comment:
It would be good to test more than order 1 elements here, as lots of elements have trivial transformations for lowest order but not for higher order.